shopyo/shopyoapi/module.py

```diff
@@ -40,3 +40,6 @@ def redirect_url(self, url, **kwargs):
 
     def context(self):
         return copy.deepcopy(self._context)
+
+    def method(self, methodname):
+        return '{}.{}'.format(self.info['module_name'], methodname)
```

Comment:
Since we are no longer supporting python < 3.6, we should use the more readable format string (`f"{self.info['module_name']}.{methodname}"`? Just a suggestion.

Reply:
Noted